.github/workflows/test-modified.yml

```diff
@@ -25,7 +25,10 @@ jobs:
       run: |
         which python
         nvidia-smi
+        rm -rf ../../MONAI/MONAI
         python -m pip install --upgrade pip wheel
+        pip uninstall -y monai
+        pip uninstall -y monai
```

Comment:
Hi @wyli , same question as https://github.com/Project-MONAI/tutorials/pull/1074#event-7907099396

Reply:
Thanks, please review #1074 first, I'll address the merging conflict here once 1074 is done